FND/Parties/Parties.rdf

```diff
@@ -113,26 +96,23 @@
 	
 	<owl:Class rdf:about="&fibo-fnd-pty-pty;IndependentParty">
 		<rdfs:subClassOf rdf:resource="&fibo-fnd-aap-agt;AutonomousAgent"/>
-		<rdfs:label>independent party</rdfs:label>
-		<owl:disjointWith rdf:resource="&fibo-fnd-pty-rl;AgentInRole"/>
-		<skos:definition>any person or organization</skos:definition>
-	</owl:Class>
-	
-	<owl:Class rdf:about="&fibo-fnd-pty-pty;OrganizationMember">
-		<rdfs:subClassOf rdf:resource="&fibo-fnd-pty-pty;PartyInRole"/>
 		<rdfs:subClassOf>
 			<owl:Restriction>
-				<owl:onProperty rdf:resource="&fibo-fnd-pty-rl;isPlayedBy"/>
-				<owl:someValuesFrom>
-					<owl:Restriction>
-						<owl:onProperty rdf:resource="&lcc-lr;isMemberOf"/>
-						<owl:allValuesFrom rdf:resource="&fibo-fnd-org-org;Organization"/>
-					</owl:Restriction>
-				</owl:someValuesFrom>
+				<owl:onProperty rdf:resource="&fibo-fnd-pty-pty;hasMailingAddress"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
 			</owl:Restriction>
 		</rdfs:subClassOf>
-		<rdfs:label>organization member</rdfs:label>
-		<skos:definition>identifies an organization or someone that plays a membership role in some organization</skos:definition>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-fnd-plc-adr;hasAddress"/>
+				<owl:onClass rdf:resource="&fibo-fnd-plc-adr;PhysicalAddress"/>
+				<owl:minQualifiedCardinality rdf:datatype="&xsd;nonNegativeInteger">0</owl:minQualifiedCardinality>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label>independent party</rdfs:label>
+		<owl:disjointWith rdf:resource="&fibo-fnd-pty-rl;AgentInRole"/>
+		<skos:definition>any person or organization</skos:definition>
```

Comment:
Need to say something about the meaning of "independent". Is a baby independent, or a wholly owned subsidiary?

Reply:
> Need to say something about the meaning of "independent". Is a baby independent, or a wholly owned subsidiary?

That name came from an IBM model years ago - MB used it to appease several FCT members at the time who were also attempting to integrate FIBO with that model.  It's a legacy thing as far as I'm concerned at this point, where 'party' would do, but changing THAT would really have far reaching consequences.